app/errors.js

```diff
@@ -1,7 +1,16 @@
+var log = require('bole')('error-handler');
+var sentry = require('raven');
 var url_for = require('./routes').url_for;
 
 
+process.on('unhandledRejection', function (error) {
+  log.error(error.message);
+  sentry.captureException(error);
+});
+
+
 module.exports = function (err, req, res, next) {
+  sentry.captureException(err);
```

Comment:
How does this line work? Will this send the exception to Sentry straight away?

Reply:
Yes, this is the global error handler middleware, so it sends the exception to Sentry when an error is passed up the request handler chain.